tethysapp/historical_validation_tool_colombia/templates/historical_validation_tool_colombia/home.html

```diff
@@ -66,42 +66,45 @@
         <div class="modal-body">
           <!-- Nav tabs -->
           <ul class="nav nav-tabs" role="tablist">
-            <li role="presentation" class="active"><a id="hydrographs_tab_link" href="#hydrographs" aria-controls="hydrographs" role="tab" data-toggle="tab">Hydrographs</a></li>
-            <li role="presentation"><a id="visualAnalysis_tab_link" href="#visualAnalysis" aria-controls="visualAnalysis" role="tab" data-toggle="tab">Visual Analysis</a></li>
-            <li role="presentation"><a id="metricsReport_tab_link" href="#metricsReport" aria-controls="metricsReport" role="tab" data-toggle="tab">Metrics Report</a></li>
-            <li role="presentation"><a id="forecast_tab_link" href="#forecast" aria-controls="forecast" role="tab" data-toggle="tab">Forecast</a></li>
+            <li role="presentation" class="nav-item active"><a class="nav-link" id="hydrographs_tab_link" href="#hydrographs" aria-controls="hydrographs" role="tab" data-bs-toggle="tab">Hydrographs</a></li>
```

Comment:
The `active` class should be moved to the `<a>` element. Fix all instances in all files.